samples/client/petstore/cpp-qt5/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(cpp-qt5-petstore)
+set(CMAKE_VERBOSE_MAKEFILE ON)
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC -std=c++14 -Wall -Wno-unused-variable")
+
+find_package(Qt5Core REQUIRED)
+find_package(Qt5Network REQUIRED)
+find_package(Qt5Test REQUIRED)
+
+file(GLOB SRCS
+    ${CMAKE_CURRENT_SOURCE_DIR}/PetStore/*.cpp
+)
+
+include_directories(
+    ${Qt5Core_INCLUDE_DIRS}
+    ${Qt5Network_INCLUDE_DIRS}
+    ${Qt5Test_INCLUDE_DIRS}
+    ${CMAKE_CURRENT_SOURCE_DIR}/client
+)
+
+add_subdirectory(client)
+add_executable(${PROJECT_NAME} ${SRCS})
+target_link_libraries(${PROJECT_NAME} PRIVATE Qt5Core Qt5Network Qt5Test ssl crypto client)
```

Comment:
I meant changing from `Qt5Core` to `Qt5::Core`, and so on. Why?
Because of changes to CMake 3 - these new targets with `::` also export include paths, so you'll only need to link with `Qt5::Core` and you'll be able to include anything from it, without additionaly using `target_include_directories`